### PR TITLE
some corrections related to font size

### PR DESCRIPTION
**Pull request by naina66406**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--world-bank--aemsites.hlx.live/
- After: https://home-bugs--world-bank--aemsites.hlx.live/

### CODE REVIEW

File: blocks/columns/columns.css [hunk 1]
```diff
@@ -65,7 +65,7 @@
 
 .bg-primary-blue-90 .columns.columns-transform-teaser p:nth-child(3) a {
   background-color: var(--white);
-  color: var(--accent-blue-primary-90);
+  color: var(--primary-blue-70);
```
Comment (thomsebastin) [inline blocks/columns/columns.css:68]: Any reason for reverting to old color variable?

**Review by thomsebastin — CHANGES_REQUESTED** [review]

File: blocks/columns/columns.css (hunk 1 above)
Comment (naina66406) [reply blocks/columns/columns.css:68]: Color code for --accent-blue-primary-90 variable is '#012740'  whereas color code in figma is '#004370' so changed the variable @thomsebastin

Comment (thomsebastin) [reply blocks/columns/columns.css:68]: But isn't there a new variable equivalent for that colour? Can you please check?

Comment (naina66406) [reply blocks/columns/columns.css]: changed into accent-blue-primary-70 @thomsebastin.
pls check

File: blocks/research-publications-cards/research-publications-cards.css [hunk 2]
```diff
@@ -6,20 +6,22 @@
...
   align-items: center;
-  margin: 0 0 var(--spacing-xlarge);
+  margin: var(--none) var(--none) var(--spacing-xlarge);
 }
 
 .cards-container .rp-card .text-content .card-btn p{
-  height: 46px;
-  display: flex;
-  align-items: center;
+  padding: 17px var(--none) var(--xxsmall);
```
Comment (thomsebastin) [inline blocks/research-publications-cards/research-publications-cards.css:13]: should be `--spacing-none` and `--spacing--xxsmall`

**Review by thomsebastin — CHANGES_REQUESTED** [review]
Please update these as well so that we can merge.

**Comment by naina66406** [comment]
Incorporate these changes. Pls check @thomsebastin

**Review by mkbansal1 — APPROVED** [review]

**Review by thomsebastin — APPROVED** [review]

**Pull request merged**